src/backends/perf_pt/mod.rs

```diff
@@ -403,6 +403,17 @@ impl Default for PerfPTThreadTracer {
     }
 }
 
+impl Drop for PerfPTThreadTracer {
+    fn drop(&mut self) {
+        if self.state == TracerState::Started {
+            // If we haven't stopped the tracer already, stop it now.
+            if let Some(err) = self.stop_tracing().err() {
```

Comment:
Is this `self.stop_tracing().unwrap()` in disguise?

Reply:
Hah, yeah, I think it is. Why write one line when you can write three?

Reply:
OK, let's simplify it then :)

Reply:
Fixed in 833fba8.